schematics/install/index.ts

```diff
@@ -38,26 +40,74 @@ function addDependenciesAndScripts(): Rule {
   };
 }
 
+function updateJsonFile<T>(
+  host: Tree,
+  path: string,
+  callback: UpdateJsonFn<T>
+): Tree {
+  const source = host.read(path);
+  if (source) {
+    const sourceText = source.toString('utf-8');
+    const json = parseJson(sourceText);
+    callback((json as {}) as T);
+    host.overwrite(path, JSON.stringify(json, null, 2));
+  }
+  return host;
+}
+const applyProjectName = (projectName, host) => {
+  if (projectName) {
+    let nestCliFileExists = host.exists('nest-cli.json');
+
+    if (nestCliFileExists) {
+      updateJsonFile(
+        host,
+        'nest-cli.json',
+        (optionsFile: Record<string, any>) => {
+          if (optionsFile.projects[projectName].compilerOptions) {
+            optionsFile.projects[projectName].compilerOptions = {
+              ...optionsFile.projects[projectName].compilerOptions,
+              ...{
+                webpack: true,
+                webpackConfigPath: `apps/${projectName}/src/webpack.config.js`
+              }
+            };
```

Comment:
Is this needed? I thought CLI will automatically load the `webpack.config.js` (if present) when it's located in the root directory of the project (even in a monorepo) - note it shouldn't be a `src` dir

Reply:
I'm going to check this and see if this is needed, thanks for that.

Reply:
I did some test and we can indeed place the `webpack.config` on the root of the project and just use the flags to use the custom `webpack.config`. Is this the solution you are proposing? 
Should we then disable the `webpack` settings for other projects where the `azure` package is referenced to? Because otherwise we always will use the custom `webpack` config if I'm correct.

Reply:
Just to clarify: what do you mean by the "root of the project"?

Currently, the `webpack.config.js` is located inside the `apps/projectTwo/src` directory.
I think that once you move this file one level lower (to the `apps/projectTwo`), it should be loaded automatically (without explicitly specifying it) unless something is wrong with the CLI :D

Reply:
Hm, I was a bit unclear so I made a repo to demonstrate it. In the repo, I made a monorepo setup with 2 apps. `app2` is the one I'm using. I also added the `webpack` flag specific in the [nest-cli.json](https://github.com/Jefiozie/nest-webpack-test/blob/master/nest-cli.json#L26) so that I'm sure it should use `webpack`. In the [`webpack.config.js`](https://github.com/Jefiozie/nest-webpack-test/blob/master/apps/app2/webpack.config.js) I placed a `console.log` just to see if it runs it.

[Here](https://github.com/Jefiozie/nest-webpack-test/tree/master/apps/app2) I placed the `webpack.config.js` under the root of the project (not in `src`). When I run `nest build app2` it doesn't use it.

[Here](https://github.com/Jefiozie/nest-webpack-test/tree/webpack-in-root) I placed the `webpack.config.js` in the root of the monorepo and now it uses it.

Did I set it up correctly or am I doing something wrong?

Reply:
Gotcha. So I think what you have now (specifying the `webpackConfigPath` etc) is fine then (just move the `webpack.config.js` one level lower (e.g., `apps/projectTwo`) - outside of the `src` dir)

Reply:
This is resolved with my latest commit